besser/bot/library/event/event_library.py

```diff
@@ -59,3 +59,24 @@ def variable_matches_operation(session: 'Session', event_params: dict) -> bool:
     operation: Callable[[Any, Any], bool] = event_params['operation']
     current_value: Any = session.get(var_name)
     return operation(current_value, target_value)
+
+
+def file_received(session: 'Session', event_params: dict) -> bool:
+    """
+    This event only returns True if the session variable "file" was set to true.
```

Comment:
I think this is wrong. "file" is not the boolean value, it is "file_received". Moreover, this event returns true if the file type matches the expected file types as well